ClientApp/viking-vault/src/AdminDashboard/AttachCardModal.tsx

```diff
@@ -120,12 +120,17 @@ class AttachCardForm extends React.Component<any, IFormState> {
     }
 
     private sendDataAndShowResponse = async () => {
+        const token = sessionStorage.getItem("Authentication-Token");
+        if(token === null) {
+            return;
+        }
         const card = this.getCard(this.props.userId);
         fetch(constants.baseUrl+"attach", {
```

Comment:
You could extract constants.baseUrl+"attach" in a constant, and name it accordingly